Mathlib/AlgebraicGeometry/Stalk.lean

```diff
@@ -262,8 +262,17 @@ def stalkClosedPointTo :
   f.stalkMap (closedPoint R) ≫ (stalkClosedPointIso R).hom
 
 instance isLocalHom_stalkClosedPointTo :
-    IsLocalHom (stalkClosedPointTo f) :=
-  inferInstanceAs <| IsLocalHom (f.stalkMap (closedPoint R) ≫ (stalkClosedPointIso R).hom)
+    IsLocalHom (stalkClosedPointTo f).hom :=
+  inferInstanceAs <| IsLocalHom (f.stalkMap (closedPoint R) ≫ (stalkClosedPointIso R).hom).hom
+
+/-- Copy of `isLocalHom_stalkClosedPointTo` which unbundles the comm ring.
+
+Useful for use in combination with `CommRingCat.of K` for a field `K`.
+-/
+instance isLocalHom_stalkClosedPointTo' {R : Type u} [CommRing R] [IsLocalRing R]
```

Comment:
This instance suggests we don't always see correctly through structure eta when looking for instances... Is that something that we can control?

Reply:
It needs again 3 `no_index` to fix it:
```
instance isLocalHom_stalkClosedPointTo :
    @IsLocalHom no_index _ no_index _ no_index _ _ _ _ (stalkClosedPointTo f).hom :=
  inferInstanceAs <| IsLocalHom (f.stalkMap (closedPoint R) ≫ (stalkClosedPointIso R).hom).hom
```
When inspecting the discr tree keys for both instances, ignoring the additional `CommRingCat.mk`s, the noticeable difference is: `(closedPoint _.1 _ _)` in the un-primed version vs `(closedPoint _ _ _)` in the primed version. Again I don't really understand what this `_.1` means.